fastlane_core/lib/fastlane_core/test_parser.rb

```diff
@@ -55,7 +66,7 @@ def initialize(path)
 
     # Returns the JUnit report as String
     def to_junit
-      JunitGenerator.new(self.data).generate
+      Fastlane::JUnitGenerator.new(self.data).generate
```

Comment:
Don't know if this even worked before since the class is called `JUnitGenerator`, not `JunitGenerator`